src/rep.rs

```diff
@@ -5,6 +5,47 @@ use chrono::{DateTime, Utc};
 use serde::{Deserialize, Serialize};
 use std::collections::HashMap;
 
+#[derive(Serialize, Deserialize, Debug)]
+#[serde(untagged)]
+/// Represents a response chunk from Docker api when building, pulling or importing an image.
+pub enum ImageBuildChunk {
```

Comment:
I'm also considering if this is the best name. Maybe `ImageStreamChunk` or something else?